plugins/chat/assets/javascripts/discourse/components/chat-channel-unread-indicator.gjs

```diff
@@ -11,15 +11,11 @@ export default class ChatChannelUnreadIndicator extends Component {
   get showUnreadIndicator() {
     return (
       this.args.channel.tracking.unreadCount > 0 ||
-      // We want to do this so we don't show a blue dot if the user is inside
-      // the channel and a new unread thread comes in.
-      (this.args.channel.isCategoryChannel &&
-        this.chat.activeChannel?.id !== this.args.channel.id &&
```

Comment:
It seems like activeChannel is always `null`, so I removed this since it isn't doing anything,